File: os2skoledata-azure-ad-sync/src/main/java/dk/digitalidentity/service/model/DBGroup.java
package dk.digitalidentity.service.model;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DBGroup {
	private long databaseId;
	private String groupName;
	private String groupId;
	private String groupLevel;
	private String institutionNumber;
	private String institutionName;
	private String azureSecurityGroupId;
	private boolean institutionLocked;
	private int startYear;
	private String azureTeamId;
	private String line;
}

File: os2skoledata-google-workspace-sync/Services/OS2skoledata/Model/DBGroup.cs
using System;

namespace os2skoledata_google_workspace_sync.Services.OS2skoledata.Model
{
    public class DBGroup
    {
        public long DatabaseId { get; set; }
        public string GroupName { get; set; }
        public string GroupId { get; set; }
        public string GroupLevel { get; set; }
        public string InstitutionNumber { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionAbbreviation { get; set; }
        public string InstitutionGoogleWorkspaceId { get; set; }
        public bool InstitutionLocked { get; set; }
        public string GoogleWorkspaceId { get; set; }
        public string StudentInstitutionGoogleWorkspaceId { get; set; }
        public string EmployeeInstitutionGoogleWorkspaceId { get; set; }
        public string DriveGoogleWorkspaceId { get; set; }
        public string GroupGoogleWorkspaceEmail { get; set; }
        public string GroupOnlyStudentsGoogleWorkspaceEmail { get; set; }
        public int StartYear { get; set; }
        public string CurrentYearGWGroupIdentifier { get; set; }
        public string CurrentYearGWFolderIdentifier { get; set; }
    }
}